### ISSUE TITLE
Enhancement: A way to export only the schema/indices and not the data but be able to import it normally too?

### ISSUE THREAD

**Issue by zifnab87** [orientechnologies/orientdb#4592]
It would be nice if I was able to extract the schema of my database plus indices without the data. Is this currently possible? I found this: http://stackoverflow.com/questions/18258272/exporting-the-orientdb-database-structure-in-a-command-style-format but I would prefer to be able to use the result through normal import/export. 

Thanks

**Comment by lvca** [comment]
@zifnab87 the output should be compatible with `import` command. Have you tried it?

**Comment by zifnab87** [comment]
well the output also contains data - I would like to be able to extract just the schema and indices.. output definitely is compatible with import already

**Comment by nagarajasr** [comment]
have you already tried this:
```
-excludeAll exclude everything. This is useful to export only few things. Instead of exclude all the feature it's much easier exclude all, and include what you're interested. Example: "-excludeAll -includeSchema=true" to export the schema only. Available since v1.7.
-includeIndexDefinitions includes or not database's index definition
```

**Comment by zifnab87** [comment]
@nagarajasr thanks! That is what I was searching for! closing
